addons/calendar_sms/tests/test_calendar_sms.py

```diff
@@ -1,27 +1,78 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from datetime import datetime
+from datetime import datetime, timedelta
+from odoo import fields
+from odoo.addons.sms.tests.common import SMSCase
+from odoo.tests.common import TransactionCase
 
-from odoo.tests.common import SingleTransactionCase
-
-
-class TestCalendarSms(SingleTransactionCase):
+class TestCalendarSms(TransactionCase, SMSCase):
 
     @classmethod
     def setUpClass(cls):
         super(TestCalendarSms, cls).setUpClass()
 
+        now = fields.datetime.now()
+
         cls.partner_phone = cls.env['res.partner'].create({
             'name': 'Partner With Phone Number',
-            'phone': '0477777777',
+            'phone': '+32477777777',
```

Comment:
FYI just setting the country should be sufficient, you just need to use `phone_sanitized` in the assert below as it will be formatted based on that.

This is fine too though